src/Validation/ConstraintViolation.php

```diff
@@ -45,10 +45,7 @@ public function setSource( string $source ) {
 		$this->source = $source;
 	}
 
-	/**
-	 * @return mixed|null
-	 */
-	public function getConstraint() {
+	public function getConstraint(): string {
```

Comment:
Ugh... so this means we do not have a single test that executes this code path?!